Makefile

```diff
@@ -0,0 +1,27 @@
+PG_CONFIG ?= $(shell which pg_config)
+DISTNAME = vchord
+# Extract DISTVERSION from vchord.control
+CONTROL_FILE = vchord.control
+DISTVERSION := $(shell sed -n "s/^[[:space:]]*default_version[[:space:]]*=[[:space:]]*'\([^']*\)'/\1/p" $(CONTROL_FILE))
+
+all: package
+
+# Install vchord into the PostgreSQL cluster identified by pg_config.
+.PHONY: install
+install:
+    @PGRX_PG_CONFIG_PATH="$(PG_CONFIG)" cargo make install -i ./build/raw
+
+# Build vchord for the PostgreSQL cluster identified by pg_config.
+.DEFAULT_GOAL: package
+package:
+    @PGRX_PG_CONFIG_PATH="$(PG_CONFIG)" cargo make build -o ./build/raw
```

Comment:
Not this command.

https://github.com/tensorchord/VectorChord/blob/ce6ce82b7cbcfef5b5326adbbbc91bb42b1ce1d5/crates/make/src/main.rs#L34

But it looks better, just rename `package` to `build` in code.